tests/python/rest_api/test_requests.py

```diff
@@ -242,6 +245,42 @@ def test_can_use_simple_filter_for_object_list(
 
         return super()._test_can_use_simple_filter_for_object_list(simple_filter, values)
 
+    def test_list_requests_when_there_is_job_with_non_regular_or_corrupted_meta(
+        self, jobs: Container, admin_user: str, request: pytest.FixtureRequest
+    ):
+        job = next(iter(jobs))
+
+        export_job_dataset(admin_user, save_images=True, id=job["id"], download_result=False)
+        export_job_dataset(admin_user, save_images=False, id=job["id"], download_result=False)
+
+        with make_api_client(admin_user) as api_client:
+            background_requests, response = api_client.requests_api.list(_check_status=False)
+            assert response.status == HTTPStatus.OK
+            assert 2 == background_requests.count
+
+            corrupted_job, normal_job = background_requests.results
+
+            remove_meta_command = f'redis-cli -e HDEL rq:job:{corrupted_job["id"]} meta'
```

Comment:
Does it make sense to check absence of individual meta fields as well?

Reply:
Oh, I see, no I think if the `user` field exists in the meta, then its structure must be as we expect in `BaseRQMeta`